libs/logger/src/logging.interceptor.ts

```
@@ -7,41 +7,50 @@ import Logger, { LoggerKey } from './logger.interface';
 import { ClsService } from 'nestjs-cls';
 import { v4 } from 'uuid';
 
-const isNullUndefinedOrEmpty = (obj: any): boolean => obj === null || obj === undefined || (typeof obj === 'object' && Object.keys(obj).length === 0);
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const isNullUndefinedOrEmpty = (obj: any): boolean =>
+  null === obj || obj === undefined || ('object' === typeof obj && 0 === Object.keys(obj).length);
```

ESLint

[error] Expected no linebreak before this expression.

(implicit-arrow-linebreak)